packages/inquirer/lib/prompts/base.js

```diff
@@ -1,5 +1,5 @@
-import defaults from 'lodash/defaults.js';
-import clone from 'lodash/clone.js';
+import defaults from 'lodash.defaults';
+import clone from 'lodash.clone';
```

Comment:
I think we could remove those with modern JS

Reply:
possibly just `{...defaults, ...options}` and `{...cloneMe}`?

Reply:
Yeah let's do that, and I don't think you'll need to clone in this context at all.